csharp-basics/exercises/Tests/ScooterRentalService/ScooterRentalService.Tests/ScooterServiceTest.cs

```diff
@@ -0,0 +1,196 @@
+﻿namespace ScooterRentalService
+{
+    [TestClass]
+    public class ScooterServiceTests
+    {
+        private ScooterService _service;
+
+        [TestInitialize]
+        public void Setup()
+        {
+            _service = new ScooterService();
+        }
+
+        [TestMethod]
+        public void GivenValidScooterIdAndPrice_WhenAddScooter_ThenScooterIsAdded()
+        {
+            var service = new ScooterService();
+            string testScooterId = "testId";
+            decimal testPricePerMinute = 1.0m;
+
+            service.AddScooter(testScooterId, testPricePerMinute);
+
+            var addedScooter = service.GetScooterById(testScooterId);
```

Comment:
testam jābūt max izolētam. :) un ja testē GetScooterById metodi tad nevajadzētu tajā pašā testā lietot AddScooterMetodi. :) 
Scooter service vajadzētu to listu dot no ārpuses. :)